regression/pages/studio/settings_studio.py

```diff
@@ -0,0 +1,53 @@
+"""
+Course Schedule and Details Settings page.
+"""
+from edxapp_acceptance.pages.studio.settings import SettingsPage
+from regression.pages import UPLOAD_FILE_DIR
+
+from regression.pages.studio.utils import (
+    get_course_key,
+    click_css_with_animation_enabled
+)
+from regression.pages.studio import BASE_URL
+
+
+class SettingsPageExtended(SettingsPage):
+    """
+    Course Schedule and Details Settings page.
+    """
+
+    @property
+    def url(self):
+        """
+        Construct a URL to the page within the course.
+        """
+        course_id = get_course_key(self.course_info)
+        return BASE_URL + "/" + self.url_path + "/" + unicode(course_id)
```

Comment:
Is `unicode` appropriate here? I'm just thinking if it's a url, it'll probably be url encoded, which does not include all unicode.